ontology/spdx-ontology.context.json

```diff
@@ -0,0 +1,422 @@
+{
+  "@context" : {
+    "spdx" : "http://spdx.org/rdf/terms#",
+    "rdf" : "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "owl" : "http://www.w3.org/2002/07/owl#",
+    "doap" : "http://usefulinc.com/ns/doap#",
+    "rdfpointer" : "http://www.w3.org/2009/pointers#",
+    "rdfs" : "http://www.w3.org/2000/01/rdf-schema#",
+    "licenseConcluded" : {
+      "@id" : "spdx:licenseConcluded",
+      "@type" : "spdx:AnyLicenseInfo"
+    },
+    "referencesFile" : {
+      "@id" : "spdx:referencesFile"
+    },
+    "licenseException" : {
+      "@id" : "spdx:licenseException",
+      "@type" : "spdx:LicenseException"
+    },
+    "referenceType" : {
+      "@id" : "spdx:referenceType",
+      "@type" : "spdx:ReferenceType"
+    },
+    "packageVerificationCode" : {
+      "@id" : "spdx:packageVerificationCode",
+      "@type" : "spdx:PackageVerificationCode"
+    },
+    "dataLicense" : {
+      "@id" : "spdx:dataLicense",
+      "@type" : "spdx:AnyLicenseInfo"
+    },
+    "member" : {
+      "@id" : "rdfs:member"
+    },
+    "members" : {
+      "@id" : "spdx:members",
+      "@type" : "spdx:AnyLicenseInfo",
+      "@container" : "@set"
+    },
+    "algorithm" : {
+      "@id" : "spdx:algorithm",
+      "@type" : "spdx:ChecksumAlgorithm"
+    },
+    "relatedSpdxElement" : {
+      "@id" : "spdx:relatedSpdxElement",
+      "@type" : "spdx:SpdxElement"
+    },
+    "licenseInfoFromFiles" : {
+      "@id" : "spdx:licenseInfoFromFiles",
+      "@type" : "spdx:SimpleLicensingInfo"
+    },
+    "licenseInfoFromFiless" : {
```

Comment:
@goneall Looks like there's an accidental double plural here ("Filess"). I note that there's a single-plural "licenseInfoFromFiles" just above it, can these be merged?

Reply:
This is an artifact of the automatic adding of plurals for the JSON format.  This one happens to already be plural.

I'll need to add a special check for this in the tool that generates both the JSON format and the files.